faults.csv

```diff
@@ -1,63 +1,63 @@
+BSO,BSOIC Tripped Chain A,BSOIC tripped,Wait for fault to clear,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyA,1,,2,BSOIC Tripped
+BSO,BSOIC Tripped Chain B,BSOIC tripped,Wait for fault to clear,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyB,1,,2,BSOIC Tripped
+PPS,Not Ready for Beam Chain A,PPS: West Global not in Ready for Beam,Wait for fault to clear,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyA,1,,2,Not Ready for Beam
+PPS,Not Ready for Beam Chain B,PPS: West Global not in Ready for Beam,Wait for fault to clear,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyB,1,,2,Not Ready for Beam
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a. rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,
+HWI,Hardware Initialization running,Hardware Initialization (ak.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,
+HWI,Hardware Initialization running,Hardware Initialization (a.k.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,Hardware Initialization needed or running
+NRP,NIRP or individual SSA interlock,NIRP or other individual SSA interlocks like PPS,"If the realtime status is OK (check legend), reset interlocks. If not, contact NIRP system experts.",SSA,PYDM,$PYDM/nirp/nirp_Lband_diagnostic_graphical.ui,"{""AREA"": {LINAC}, ""CM"": {CRYOMODULE}, ""RFNAME"": ""CM{CRYOMODULE}""}",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,NRP_PRMT,1,,2,NIRP or individual SSA interlock
+SSP,SSA Permit,SSA permit to turn on faulted,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,SSA_LTCH,0,,2,SSA Permit
+SSA,SSA Faulted,SSA not on,,SSA,EDM,$EDM/llrf/rf_srf_ssa_{cm_OR_hl}.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,FaultSummary.SEVR,,2,2,SSA Faulted
+BCS,BCS LLRF Drive Fault,BCS fault is interrupting LLRF drive (only affects CM01 in practice),Wait for fault to clear,CM,EDM,$EDM/bcs/ops_lcls2_bcs_main.edl,,,ACCL:{LINAC}:{CRYOMODULE}00:,BCSDRVSUM,0,,2,BCS LLRF Drive Fault

```

Comment:
Not sure if 'wait for fault to clear' is the best instruction. That sounds like it would go away on it's own. How about 'Resolve BCS fault'?